creds.yml

```diff
@@ -0,0 +1,9 @@
+$ANSIBLE_VAULT;1.1;AES256
```

Comment:
git ignore this file

Reply:
Yes, for sure. Actually I have added the file into .gitignore file in base git directory, let me check why it is still getting pushed.

Reply:
Yes, for sure. Actually I have added the file into .gitignore file in base git directory, let me check why it is still getting pushed.

Reply:
.gitignore only ignores files that are not part of the repository yet. Since I already added it in the past, its changes will still be followed up, as a result I ran `git rm --cached` on creds.yml.

Reply:
As of now, it should be tracked again as per the .gitignore configuration already there in the repo.